bin/wm_append_diffusion_measures_across_subjects.py

```diff
@@ -101,13 +101,13 @@ def main():
     fields = [col.strip() for col in stats.columns]
     fields = fields[1:]
     print()
-    print('Fiber cluster diffusion measures per subject: (n=%d): ' % len(fields))
+    print(f'Fiber cluster diffusion measures per subject: (n={len(fields)}): ')
     print(fields)
 
     clusters = stats.to_numpy()[:, 0]
     clusters = [os.path.basename(filepath).replace('.vtp', '').replace('.vtk', '').strip() for filepath in clusters]
     print()
-    print('Fiber clusters per subject: (n=%d): ' % len(clusters))
+    print(f'Fiber clusters per subject: (n={len(clusters)}): ')
 
     if len(clusters) == 800 and len(tracts) == 73:
         comm_clusters = [3, 8, 33, 40, 46, 52, 56, 57, 62, 68, 69, 73, 86, 91, 109, 110, 114, 142, 144, 145, 146, 159, 163, 250, 251, 252, 257, 262, 263, 268, 271, 305, 311, 314, 322, 330, 334, 338, 342, 350, 363, 371, 375, 403, 410, 437, 440, 448, 456, 465, 468, 475, 484, 485, 488, 519, 522, 525, 543, 545, 549, 557, 576, 577, 582, 587, 591, 597, 601, 614, 620, 623, 627, 633, 645, 653, 658, 663, 670, 677, 683, 702, 770, 781]
```

Comment:
These should not be hard-coded here; they should be in a separate file so that
- they can be located easily.
- they can be reused by other scripts, if necessary.
- if they need to be changed, this can be done easily in a place that contains only data-/atlas-related labels, and does not pollute the code.